MetaCell/salk-interactive-atlas
# Feature/213

Closes #213

Implemented solution: 
- Updates geppetto's version
- Updates other few packages versions to coop with react 17
- Updates io mapping pdf

How to test this PR: 
- Check if application builds
- Verify if I/O mapping pdf is the new one (not the ephys placeholder)
![image](https://user-images.githubusercontent.com/19196034/196158467-b4633617-cf55-41b6-8e70-92d994671011.png)


### Sanity checks:

### Breaking changes (select one):

### Possible deployment updates issues (select one):

### Test coverage (select one):

### Documentation (select one):
